added test file and main file

benwilburn opened the pull request.

benwilburn committed:
added test file and main file

benwilburn commented:
I just started the quiz, and I am in the planning stage. So all I have in the pull request is two empty files

benwilburn committed:
added txt files for users/private, public chirps, added first menu

benwilburn committed:
added menu for new_user()

CharisseL commented:
The first thing that I notice at this point is that you have begun implementing code but there are no tests for Birdyboard yet. Remember: write a few tests, then implement the code for it.

benwilburn commented:
@CharisseL I haven't written any implementing code. I am just writing the menu code. I asked Ryan and he said it was fine to do that, as long as I don't write any logic. If thats not the case, i'll just delete what I have.

CharisseL commented:
@benwilburn no, don't delete it. I should have rephrased that. You are fine doing that w/ the menu code.

benwilburn commented:
@CharisseL alright. I just wanted to clarify, based on what I was told. Thank you!

benwilburn committed:
broke out tests into separate files
wrote tests for chirp.py, users.py

benwilburn committed:
wrote first tests for conversation

benwilburn committed:
added implementation code for tests

benwilburn committed:
added secondary menu

benwilburn committed:
changed the select user to return a value instead of setting it

benwilburn committed:
successfully printed out all public chirps

and private chirps

benwilburn committed:
changed setup of my classes

benwilburn committed:
removed commented out code

benwilburn committed:
refactored birdyboard.py
it was getting confusing looking at the code without any comments and only in a few functions. So, I refactored to make it easier to read and commented each piece to specify what it was doing.

benwilburn committed:
commented a few more lines

benwilburn committed:
successfully got the private chrips working

benwilburn committed:
wrote doc strings for all methods in birdyboard

stevebrownlee commented:
@benwilburn 👍
